modules/installation/pages/server-setup.adoc

```diff
@@ -9,16 +9,17 @@ This section covers {productname} Server setup, using these procedures:
 * Synchronize the {sle} product channel from {scc}
 
 [WARNING]
-.Third Party Software
+.Installation of software on {productname} hosts
 --
 {productname} is part of the {sle} 4.0 product family and thus compatible with the software shipped with {sles}.
 
-{productname} is a complex system, and therefore installing third party software is not allowed.
+{productname} is a complex system, and therefore installing software is not allowed.
```

Comment:
My gripe with "third-party" is PackageHub: it's SUSE's, not a random third-party.

Reply:
Since we have an explicit note about packagehub now, couldn't we just leave this one stand as third-party then? Or say something like "third-party vendors, including PackageHub"?

Reply:
Do you mean having two warning notes, one about third-party and the other about PackageHub?

That was actually how I was going to write it initially but I didn't want to duplicate the "suse manager is a complex product" part. But yes, I think that'd cleaner.

Reply:
I don't think we need to mention that suma is a complex product, it's redundant information.